src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,49 @@
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.calendar {
+  display: flex;
+  width: $calendar-width;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  &__field {
+    width: $field-width;
+    height: $field-width;
+    background-color: #eee;
+    border: $border-field solid #000;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition-duration: 0.5s;
+    transition-delay: 0.5s;
+    transition-timing-function: ease-in-out;
+    transition-property: all;
```

Comment:
please say explicitly which CSS properties you want to transition, because now even changing calendar length is weirdly animated